kordac/Kordac.py

```diff
@@ -2,22 +2,35 @@
 import mdx_math
 from kordac.KordacExtension import KordacExtension
 
-class Kordac():
+class Kordac(object):
+    """A converter object for converting markdown
+    to HTML"""
 
-    def run(self, md_string):
-        self.heading = 'I am a heading'
+    def run(self, markdown_string):
```

Comment:
I used `md_string` to match `html_string`....which I think has changed since to just `html`, so whatever works for you, personally I prefer to specify that it's a string, since `html` could also be read as a collection of ..."objects" (?) (recall the `etree`)